docs/en/sdk-manual/wechat.md

```diff
@@ -0,0 +1,323 @@
+# Clickstream WeChat Miniprogram SDK
```

Comment:
Typo: In word 'Miniprogram' , refer to https://developers.weixin.qq.com/miniprogram/en/dev/framework/

Reply:
Synced offline. We'll use "WeChat **Miniprogram** SDK" as the SDK name and use "WeChat Mini Program" in the docs and code comments.